Mayflower v10  rush monorepo (#1195)
massgov/mayflower: README.md

[react-npm-badge]: https://img.shields.io/npm/v/@massds/mayflower-react.png
[npm]: https://www.npmjs.com/package/@massds/mayflower-react

## Monorepo Package Management

> This PR adds support for the creation of a monorepo using [Rush](https://rushjs.io/) and [PNPM](https://pnpm.js.org/). To install both globally, run the following command:

Reviewer: ```suggestion
This monorepo is managed using [Rush](https://rushjs.io/) and [PNPM](https://pnpm.js.org/). To install both globally, run the following command:
```
